src/components/Approval/Approval.tsx

```diff
@@ -81,35 +81,8 @@ export const Approval = () => {
         approvalInterval.current && clearInterval(approvalInterval.current)
         return history.push('/trade/input')
       }
-
       approvalInterval.current && clearInterval(approvalInterval.current)
-      if (!sellAsset.amount) return
-      if (!quote) return
-
-      fnLogger.trace({ fn: 'buildQuoteTx' }, 'Building Quote...')
-      let result
-      try {
-        result = await buildQuoteTx({
-          wallet,
-          sellAsset: quote?.sellAsset,
-          buyAsset: quote?.buyAsset,
-          amount: sellAsset?.amount,
-        })
-
-        fnLogger.debug({ fn: 'buildQuoteTx', result }, 'Building Quote Completed')
-      } catch (e) {
-        fnLogger.error(e, { fn: 'buildQuoteTx' }, 'Building Quote Failed')
-      }
-
-      if (!result?.success && result?.statusReason) {
-        handleToast(result.statusReason)
-      }
-
-      if (result?.success) {
-        history.push({ pathname: '/trade/confirm', state: { fiatRate } })
-      } else {
-        history.push('/trade/input')
-      }
+      history.push({ pathname: '/trade/confirm', state: { fiatRate } })
```

Comment:
I know this is coming from the previous code, but can we move the routes into an enum while at it?